### DISCUSSION TITLE
Enable vulnerability detector

### DISCUSSION THREAD

**Discussion by xaled1** [wazuh/wazuh-docker#862] [Q&A] [answered]
Hi,

what is the best way to enable vulnerability detector? 

ossec.conf has is disabled:

vulnerability-detector>
    <enabled>no</enabled>

**Comment by rauldpm** [comment, marked as answer]
Hello @xaled1 

The `ossec.conf` file is a system-mounted file so changes applied outside the container should be applied inside the container, but you must restart the container for the Wazuh manager service to apply it.

For this it is necessary that you modify the configuration file in the repository that you have cloned, in the case of a single node, it would be the following file:

- https://github.com/wazuh/wazuh-docker/blob/v4.4.4/single-node/config/wazuh_cluster/wazuh_manager.conf
- https://github.com/wazuh/wazuh-docker/blob/4e7c2cf72aa29f8717cccf681bb8d64237a05460/single-node/config/wazuh_cluster/wazuh_manager.conf#L99

Keep in mind that by Docker philosophy, there is no native persistence, so if you apply certain changes inside the container without using the volume, you may lose data.

**Comment by teddytpc1** [comment]
Issue locked due to inactivity, feel free to open an issue or another discussion if necessary
